locales/pl/php-inline.json

```
@@ -92,10 +92,10 @@
     "password.symbols": "To pole musi zawierać przynajmniej jeden symbol.",
     "password.uncompromised": "Podane pole pojawiło się w wycieku danych. Proszę wybrać inną wartość pola.",
     "present": "Pole musi być obecne.",
-    "present_if": "This field must be present when :other is :value.",
-    "present_unless": "This field must be present unless :other is :value.",
-    "present_with": "This field must be present when :values is present.",
-    "present_with_all": "This field must be present when :values are present.",
+    "present_if": "Pole :attribute musi być obecne jeżeli :other ma wartość :value.",
+    "present_unless": "Pole :attribute musi być obecne, chyba że :other ma wartość :value.",
+    "present_with": "Pole :attribute musi być obecne kiedy :values jest podany/a..",
+    "present_with_all": "Pole :attribute musi być obecne kiedy :values są podane.",
```

```suggestion
    "present_if": "Pole musi być obecne jeżeli :other ma wartość :value.",
    "present_unless": "Pole musi być obecne, chyba że :other ma wartość :value.",
    "present_with": "Pole musi być obecne kiedy :values jest podany/a..",
    "present_with_all": "Pole musi być obecne kiedy :values są podane.",
```
(I don't know if it's right or not, just did it like the other keys.)